content/about/privacy.md

```diff
@@ -0,0 +1,347 @@
+---
+title: Privacy Policy
+description: ""
+authors: poorvi-vaish
+---
```

Comment:
Missing category. it should be `category: privacy` after authors
Please add `Privacy Policy | Node.js` for the description